spikes/xml/claim.xml

```diff
@@ -0,0 +1,237 @@
+<claim>
+    <claim_details>
+        <claim_uuid>2161f858-cb16-4c77-9792-28defcbc5688</claim_uuid>
+        <!-- type can be Advocate, Litigator Final, Litigator Interim, Litigator Transfer -->
+        <type>Advocate</type>
+        <!--provider_key is the key that CCR/CCLF use to identify provider-->
+        <provider_key>2161f858-cb16-4c77-9792-28defcbc5688</provider_key>
+        <!--We can either send an id that identifies the creator to CCCD, or an id used byCCRR/CCLF -->
+        <supplier_number>1B222Z</supplier_number>
+        <created_by_uuid>2161f859-cb16-4c77-9792-28defcbc5688</created_by_uuid>
+        <!--External_user_uuid identifies the advocate or litigator who is making the claim-->
+        <external_user_uuid>2161f859-cb16-4c77-9792-28defcbc5688</external_user_uuid>
```

Comment:
I wonder if we would need to send them more details, as they won't have the corresponding data for these UUIDs.